LICENSE

```diff
@@ -0,0 +1,21 @@
+The MIT License (MIT)
+
+Copyright (c) 2015 Mattias Erming
```

Comment:
add « and contributors », since there is no [CLA](https://en.wikipedia.org/wiki/Contributor_License_Agreement) as far as I know.